e2e/helpers.ts

```diff
@@ -200,6 +204,8 @@ export async function getExtensionIdByName(
       .getText();
     return text;
   } else {
+    return 'lccajkdjonnokmddpnclgcihidkeklij';
```

Comment:
I think this would need to be dynamic; each install gets a random id IIRC

Reply:
Yes the method to find the extension by Id is broken in chrome 133 so I had this hardcoded to test. I'll revert it was mistakenly committed.